buildsystem/gcc-cuda/ascentVariables.sh

```diff
@@ -7,3 +7,5 @@ source $SRCDIR/buildsystem/gcc-cuda/ascent/base.sh
 
 # Spack modules
 source $SRCDIR/buildsystem/spack/$MY_CLUSTER/modules/dependencies.sh
+
+module load spectrum-mpi/10.4.0.3-20210112
```

Comment:
```suggestion

# Spack modules
source $SRCDIR/buildsystem/spack/$MY_CLUSTER/modules/dependencies.sh
```

IMO what we could do is:
- Remove `module purge` equivalents from scripts, which might nuke other friendly things that are loaded by default
- Load spack modules, then load `base.sh`. That way we can keep all custom and weird fixes to the environment in there, and also ensure it is run last

Reply:
I was suggesting extending this to all platforms, but this is great for now. Thanks!